vars/fcosBuild.groovy

```diff
@@ -19,7 +20,8 @@ def call(params = [:]) {
         shwrap("mkdir -p ${cosaDir}")
 
         if (!params['skipInit']) {
-            shwrap("cd ${cosaDir} && cosa init https://github.com/coreos/fedora-coreos-config")
+            params['args'] = "init https://github.com/coreos/fedora-coreos-config"
+            utils.cosaCmd(params)
```

Comment:
I think it would be cleaner to separate the `params` from config/state. For readability, having the second param of `args` would be cleaner and improve the readability.  Or go-full OOO and have `cosaCmd` be an object that is created with the parameters. In this case, OOO might be worth the trouble since we're wrapping COSA. 

```
class cosaCmd {
   String CosaDir
   String User

   def run(args) {
         if(!params['user']) {
            return shwrap("cd ${CosaDir} && cosa ${args}")
         } else {
            return shwrap("cd ${CosaDir} && sudo -u ${User} cosa ${args}")
        }
    }

}

```

And then you would use:
```
def cosaCommander = new utils.cosaCmd("/srv", "builder")
cosaCommander.run("init ....")
```